BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -33,16 +33,15 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     var lastUpdated: Date?
     let maxSearchBarHeight: Int = 50
     
-    var goals : Array<GoalProtocol> = []
-    var filteredGoals : Array<GoalProtocol> = []
+    var goals : Array<Goal> = []
+    var filteredGoals : Array<Goal> = []
```

Comment:
Since we are touching these lines anyway, how about adjusting them to this more modern syntax?

```suggestion
    var goals = [Goal]()
    var filteredGoals = [Goal]()
```

or

```suggestion
    var goals: [Goal] = []
    var filteredGoals: [Goal] = []
```